### PR TITLE
Gallery: fix items count pronouncing by screen reader

### PR DESCRIPTION
**Pull request by Krijovnick**

### CODE REVIEW

File: js/ui/gallery.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ const GALLERY_INDICATOR_CLASS = GALLERY_CLASS + '-indicator';
 const GALLERY_INDICATOR_ITEM_CLASS = GALLERY_INDICATOR_CLASS + '-item';
 const GALLERY_INDICATOR_ITEM_SELECTOR = '.' + GALLERY_INDICATOR_ITEM_CLASS;
 const GALLERY_INDICATOR_ITEM_SELECTED_CLASS = GALLERY_INDICATOR_ITEM_CLASS + '-selected';
+const ITEM_CONTENT_CLASS = '.dx-item-content';
```
Comment (marker-dao) [inline js/ui/gallery.js]: It is not a class. It is a selector. Could you change a variable name or value like this:

`const ITEM_CONTENT_SELECTOR` or `'dx-item-content'`

File: js/ui/gallery.js [hunk 2]
```diff
@@ -638,25 +641,16 @@ const Gallery = CollectionWidget.inherit({
...
-
-        this._itemElements().each((function(index, item) {
-            if(this.option('selectedIndex') === index) {
-                $(item).removeClass(GALLERY_INVISIBLE_ITEM_CLASS);
-            } else {
-                $(item).addClass(GALLERY_INVISIBLE_ITEM_CLASS);
+        const selectedIndex = this.option('selectedIndex');
+        this._itemElements().each((index, item) => {
+            if(selectedIndex !== index) {
+                $(item).find(ITEM_CONTENT_SELECTOR).addClass(GALLERY_INVISIBLE_ITEM_CLASS);
```
Comment (ksercs) [inline js/ui/gallery.js:647]: does the previous selected index get this class now?

File: js/ui/gallery.js [hunk 3]
```diff
@@ -638,25 +641,16 @@ const Gallery = CollectionWidget.inherit({
...
+        });
     },
 
     _releaseInvisibleItems: function() {
-        this._itemElements()
-            .removeClass(GALLERY_INVISIBLE_ITEM_CLASS);
-
-        this._getLoopedItems()
-            .removeClass(GALLERY_INVISIBLE_ITEM_CLASS);
+        this._itemElements().find(ITEM_CONTENT_SELECTOR).removeClass(GALLERY_INVISIBLE_ITEM_CLASS);
```
Comment (ksercs) [inline js/ui/gallery.js:653]: what about looped items?

File: js/ui/gallery.js (hunk 2 above)
Comment (Krijovnick) [reply js/ui/gallery.js:647]: No, before call the '_renderItemVisibility' method always called '_releaseInvisibleItems' method that remove hidden class

File: js/ui/gallery.js (hunk 3 above)
Comment (Krijovnick) [reply js/ui/gallery.js:653]: The '_itemElements' returns all elements

**Review by alexanderPolosatov — APPROVED** [review]

**Review by marker-dao — APPROVED** [review]

**Pull request merged**